filter/src/test/java/gov/cms/ab2d/filter/ExplanationOfBenefitsTrimmerTest.java

```diff
@@ -129,8 +129,8 @@ void demo1() {
     void isPlanD() {
         ExplanationOfBenefit ePlanD = ExplanationOfBenefitsTrimmer.getBenefit(
                 EOBLoadUtilities.getEOBFromFileInClassPath("eobdata/EOB-for-Part-D-Claims.json", context));
-        assertTrue(EOBLoadUtilities.isPlanD(ePlanD));
-        assertFalse(EOBLoadUtilities.isPlanD(eobCarrier));
-        assertFalse(EOBLoadUtilities.isPlanD(null));
+        assertTrue(EOBLoadUtilities.isPartD(ePlanD));
```

Comment:
Do you need to update the variable name here for ePlanD?

Reply:
Fair enough. I'll change it.